curl-sys/lib.rs

```diff
@@ -582,6 +582,8 @@ pub const CURLOPT_PROXY_CAPATH: CURLoption = CURLOPTTYPE_OBJECTPOINT + 247;
 pub const CURLOPT_PROXY_SSLCERT: CURLoption = CURLOPTTYPE_OBJECTPOINT + 254;
 pub const CURLOPT_PROXY_SSLKEY: CURLoption = CURLOPTTYPE_OBJECTPOINT + 256;
 
+pub const CURLOPT_MAXAGE_CONN: CURLoption = 288;
```

Comment:
This should be `CURLOPTTYPE_LONG + 288`. Curl's headers can be a bit hard to grok because of the macros, but the `CURLOPT` macro actually adds an offset to the supplied number when it expands.

Reply:
Ah, thanks for sharing. I'll update that.